content/docs/composition-vs-inheritance.md

```diff
@@ -8,15 +8,15 @@ prev: lifting-state-up.html
 next: thinking-in-react.html
 ---
 
-React has a powerful composition model, and we recommend using composition instead of inheritance to reuse code between components.
+React का कम्पोजीशन मॉडल शक्तिशाली है, और हम सलाह देते है कि आप कौम्पोनॅन्ट के बीच कोड के दुबारा प्रयोग के लिए इनहेरिटेंस कि जगह कम्पोजीशन का प्रयोग करे|
 
-In this section, we will consider a few problems where developers new to React often reach for inheritance, and show how we can solve them with composition.
+इस भाग में, हम कुछ समस्याएं देखेंगे जहाँ React के नए डेवेलपर्स इनहेरिटेंस का प्रयोग करने की दिशा में जाते है, और दिखाएंगे की हम उनका समाधान कम्पोजीशन से कैसे निकाल सकते हैं|
 
-## Containment {#containment}
+## कन्टेनमेंट {#containment}
 
-Some components don't know their children ahead of time. This is especially common for components like `Sidebar` or `Dialog` that represent generic "boxes".
+कुछ कौम्पोनॅन्ट अपने चिल्ड्रन के बारे में समय से पूर्व नहीं जान पाते| यह उन कौम्पोनॅन्ट्स के लिए आम बात है, जो सामान्य "boxes" दर्शाते है, जैसे `Sidebar` एवं `Dialog`।
 
-We recommend that such components use the special `children` prop to pass children elements directly into their output:
+हम सलाह देते है कि, यह कौम्पोनॅन्ट्स विशेष `चिल्ड्रन` props का प्रयोग करते है जिनसे चिल्ड्रन एलिमेंट्स को सीधा आउटपुट में पास किया जाता है|
```

Comment:
children should not be translated

Reply:
प्रयोग करते है जिनसे -> kare jinse

Reply:
पास किया जाता है| -> karna mumkin hoga

Reply:
@saranshkataria @arshadkazmi42  We need to decide on whether to convert children or not.

Because might be there is some confusion regarding that?

Reply:
When it is in quotes, we are not changing it anywhere is what my understanding was

Reply:
@arshadkazmi42  ?

Reply:
I think, we are not translating the words which are very specific to react or which might change the meaning in translation.
I have encountered children word in alot of pages and I think we have translated it everywhere.

Reply:
@arshadkazmi42  It totally depends. I will be in favour of translating the word into Hindi.

@saranshkataria  What's your opinion?

Reply:
@arshadkazmi42 I am okay translating the word unless it is inside quotes, those get syntax highlighted I think

Reply:
@saranshkataria That also makes sense, let's keep it like that then. We should also add it in glossary

Reply:
@arshadkazmi42 @saranshkataria 

I am updating my PR accordingly i.e. Leaving children in English itself everywhere for uniformity. Just pushing code. 

Kindly move forward with further reviews.

Reply:
@apoorvtomar2222 cool.
@saranshkataria Can you add this in glossary if you got some time? 
Otherwise, I will add it later today.